lib/screens_web/router.ex

```diff
@@ -84,6 +84,9 @@ defmodule ScreensWeb.Router do
 
       get "/:id", ScreenController, :index
       get "/:id/simulation", ScreenController, :simulation
+      post "/:id/widget", ScreenController, :widget
+      # Get not needed for application, but for testing in the browser w/o the Mercury skin
+      get "/:id/widget", ScreenController, :widget
```

Comment:
Oh I just thought of something, does the GET version of this endpoint work? Because the controller function expects `widget` to be an actual map, not a JSON string.

Phoenix has a convenience feature where it automatically decodes JSON in a POST request's body, which is why the controller function works properly for those. But I'm not sure if it does the same for JSON in a GET request's url params.

If the controller function receives a JSON string for `widget`, then `Jason.encode!(widget_data)` will double-encode it and make the frontend code fail.

```ex
iex> json_string = "{\"foo\": \"bar\"}"
iex> Jason.encode!(json_string)
"\"{\\\"foo\\\": \\\"bar\\\"}\""
```
😱

Reply:
Hmm ok. This get isn't really supposed to work on its own, but was only needed for getting the proper csrf token needed to do other testing in postman. And when the widget endpoint handler used to have the line 
```
    |> assign(:widget_data, if(params["widget"], do: Jason.encode!(params["widget"]), else: nil))
```
then the get request "succeeded" and provided the csrf header needed to set up the post requests. 

I guess I'll just remove this since it won't be useful for anyone in the future